changelogs/gql-api-stable-version.md

```diff
@@ -0,0 +1,13 @@
+---
+title: GraphQL API v1
+type: added
+---
+
+Released stable version v1 of GraphQL API.
```

Comment:
I don't know if I like `v1`. GraphQL [recommends](https://graphql.org/learn/best-practices/#versioning) not to version but evolve the API instead. So we don't plan on making a v2.

Reply:
That makes sense, I will remove the version info 👍